Module2/springhometask/src/main/java/com/epam/spring/dao/impl/StaticUserDao.java

```diff
@@ -1,5 +1,6 @@
 package com.epam.spring.dao.impl;
 
```

Comment:
class StaticUserDao not implementing related interface - EventDAO
StaticUserDao - is not good name, it should not contain "static" word

Reply:
Renamed